AWGL/auto_qc
# WGS and WES functionality, advanced download form

to test:
1. pull directory, install environment, create superuser
2. loaddata sample_sheet/fixtures/assay.json
3. loaddata sample_sheet/fixtures/referraltype.json
4. follow import of index set guide found in SOP to import sample_sheet/index_files/TSO500a_indexes.tsv (uses manage.py command)

WGS:
6. follow upload of worksheet guide from SOP to import sample_sheet/example_shire_queries/wgs_pcrfree_shire_query.csv
7. fill out worksheet clin and tech tabs, include trying HPO ID and different referral types. (if referral types don't exist you may need to populate many to many field in /admin referraltype objects)
8. download samplesheet, check formatting/content of description field etc is correct.

WES:
10. follow upload of worksheet guide from SOP to import sample_sheet/example_shire_queries/WES_shire_query.csv
11. fill out worksheet clin and tech tabs as above
12. download samplesheet and check formatting/content etc is correct.

advanced download:
14. make sure both the WGS and WES worksheet are signed off fully.
15. click the download button in one, then the advanced option and enter the alternate worksheet ID into the text box.
16. check samplesheet is as expected.
